content/aips/7CDA90A3999E-AIP-REDEEM-AFEI-FOR-DAI.md

```diff
@@ -0,0 +1,76 @@
+created: 2022-09-28
+---
+
+# Simple Summary
+
+This proposal proposes unwinding the aFEI holding for DAI and depositing the DAI into the Aave v2 Ethereum Pool DAI Reserve.
+
+# Abstract
+
+Due to recent events within the Tribe DAO community, this proposal will over time redeem all aFEI in the Ethereum Reserve Factor for FEI. The FEI holding will then be redeemed for DAI via Tribe DAO’s newly deployed SimpleFeiDaiPSM and deposited into the Aave v2 Ethereum DAI Reserve to earn yield, [1].
+
+This will reduce Aave’s exposure to FEI. 
+
+# Motivation
+
+The FEI reserve is concentrated with Aave DAO holding 94.3% of all aFEI supply, [2]. 
+
+The reserve currently supports 662,518.93 units of aFEI, with 582,449.43 units of debt and 80,069.50 units available for lenders to redeem their collateral from or for Users to borrow, [3,4]. Any User seeking to claim 80,069.50 units or less is able to do so now, but this may change as others start redeeming aFEI. 
+
+Limited FEI liquidity remains in the Reserve. Aave DAO is dependent on Borrowers repaying their debt which provides the liquidity needed for Aave to redeem the aFEI position.
+
+Once the aFEI has been redeemed for FEI, it is to be redeemed for DAI via Tribe DAO’s SimpleFeiDaiPSM. There is no redeem fee and all FEI is to be redeemed for DAI via this contract.
+
+The SimpleFeiDaiPSM is supported by Tribe DAO as means of removing FEI from circulation, [5]. The SimpleFeiDaiPSM has around $57,4M of liquidity at the time of writing, [1].
+
+# Specification
+
+The below provides a broad, high level summary of how the aFEI position held within the Reserve Factor will be managed. 
+
+Any EOA can call the swapAllAvailable function from the Swapper contract.
+The function will redeem the maximum amount of aFEI based upon available liquidity within the FEI Reserve and the Aave Mainnet Reserve Factor.
+All FEI is to be redeemed for DAI via the Tribe DAO’s SimpleFeiDaiPSM.
+DAI will be deposited in the Aave v2 Ethereum DAI Reserve.
+Aave v2 Ethereum Reserve Factor will receive the aDAI
+
+# Test Cases
+
+This proposal has been tested and peer reviewed by Bored Ghost Developing, [6].
+
+Test cases can be found at [7]
+
+Payload and Swapper can be found at [8] & [9] respectively.
+
+## Implementation
+
+The governance payload is deployed at [10], the swapper contract is deployed at [11]. 
+
+The proposal transfers aFEI from the Aave Mainnet Reserve Factor (0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c) to the payload contract via the Aave Ecosystem Controller (0x3d569673dAa0575c936c7c67c4E6AedA69CC630C) transfer function.
+
+The proposal redeems aFEI for FEI via the Lending Pool (0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9).
+
+The proposal redeems all available FEI for DAI via the Tribe DAO Fixed Price PSM (0x2A188F9EB761F70ECEa083bA6c2A40145078dfc2) sending all DAI directly to the Aave Mainnet Reserve Factor (0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c).
```

Comment:
The DAI is not being sent to the Aave Mainnet Reserve Factor right? It's being deposited for aDAI which is then being sent to the Aave Mainnet Reserve Factor.